containerbuddy/config.json

```diff
@@ -1,11 +1,12 @@
 {
-  "consul": "consul:8500",
+  "consul": "{{.CONSUL}}:8500",
+  "onStart": "/opt/containerbuddy/onstart.sh",
   "services": [
     {
-      "name": "wordpress",
+      "name": "app",
```

Comment:
Either the Compose file or in the `_env` file will work.

There's also an argument for passing the name of the service that Nginx watches as an environment var. Perhaps @tgross has an opinion on that.